src/framework/EventBusManager.cpp

```diff
@@ -16,9 +16,9 @@ namespace OpenWifi {
 		KafkaManager()->PostMessage(KafkaTopics::SERVICE_EVENTS, MicroServicePrivateEndPoint(), Msg,
 									false);
 		while (Running_) {
-			Poco::Thread::trySleep((unsigned long)MicroServiceDaemonBusTimer());
-			if (!Running_)
```

Comment:
What this code is for?

Reply:
my guess is that this is a bug fix and he used the wrong variable (after all, Running_ is never updated in the loop so it can never break out of the loop)